README.md

```diff
@@ -1,10 +1,11 @@
 # Overview
 
-This is a coding exercise to help us assess candidates looking to join the team at Condé Nast.  The test is intended to be completed at home and we expect you to spend no more than 1-2 hours working on it. We appreciate the time you spend on it and we will go through your submitted work together when you're invited to meet us face-to-face.        
+This is a coding exercise to help us assess candidates looking to join the team at Condé Nast. The task is intended to be completed at home. We appreciate the time you spend on it and we will go through your submitted work together when you're invited to meet us face-to-face.        
```

Comment:
👍 though would like to get some broader feedback (since this is the base readme)

Reply:
sure.
As mentioned, there is no way the iOS task can be implemented in two hours :-)